src/main/java/tech/stackable/gis/hbase/coprocessor/DistComp.java

```diff
@@ -10,7 +10,7 @@ public class DistComp implements Comparator<Neighbor> {
     static final Log LOG = LogFactory.getLog(DistComp.class);
     private final Point2D origin;
 
-    public DistComp(double lat, double lon) {
```

Comment:
Now that `Neighbor` has the distance field, the `DistComp` doesn't need the origin or even compute the distance in `compare()` anymore.
Something like this should suffice:
```
    @Override
    public int compare(Neighbor o1, Neighbor o2) {
        if (o1.distance < 0 || o2.distance < 0)
            LOG.warn("negative distance detected!");
        return Double.compare(o1.distance, o.distance);
    }
```

Reply:
Done in c79e71a